Scheduler/feature_scheduler/auxtel/2022/03/fbs_config_sp_poor_conditions.py

```diff
@@ -0,0 +1,187 @@
+import os
+
+import numpy as np
+
+import astropy.units as u
+
+from pathlib import Path
+
+from astropy.coordinates import Angle
+from astropy.io import ascii as astropy_ascii
+
+from rubin_sim.scheduler import basis_functions
+
+from rubin_sim.scheduler.utils import empty_observation
+from rubin_sim.scheduler.schedulers import Core_scheduler
+from rubin_sim.scheduler.surveys import Greedy_survey, FieldSurvey
+
+
+def get_basis_functions_cwfs_survey(
+    nside,
+    note,
+    time_gap_min,
+    wind_speed_maximum,
+):
+    """Get the basis functions for the CWFS survey.
+
+    This is a background survey that will activate at specific points in time
+    to make sure the telescope optics is aligned.
```

Comment:
`is` -> `are`. Please check the other new python files as well. You (understandably) seem to have done a lot of copying and pasting hehe